htdocs/map.js

```diff
@@ -174,7 +174,7 @@ MapManager.prototype.processUpdates = function(updates) {
                     self.mman.addType(update.mode);
                     self.mman.add(update.callsign, marker);
                     marker.addListener('click', function() {
-                        showMarkerInfoWindow(update.callsign, marker.pos);
+                        showMarkerInfoWindow(update.callsign, marker.position);
```

Comment:
this and the similar are fixes for handling the url (callsign and locator) params for GMaps.